### PR TITLE
[yank] release 0.9.0

### PR DESCRIPTION
**Pull request by andrrizzi**
Release 0.9.0

### CODE REVIEW

**Comment by rmcgibbo** [comment]
Congrats!

File: yank/meta.yaml [hunk 1]
```diff
@@ -21,15 +21,16 @@ requirements:
     - scipy
     - setuptools
     - netcdf4
-    - openmm ==7.0rc1 # temporarily test against stable version
+    - openmm  ==7.0.1
     - mdtraj 1.5.1 # TODO remove me when segment_id bug is fixed (this is installed through openmoltools)
```
Comment (mpharrigan) [inline yank/meta.yaml]: I think the cited bug is fixed in the latest release of mdtraj

File: yank/meta.yaml [hunk 2]
```diff
@@ -40,18 +41,19 @@ requirements:
     - scipy
     - cython
     - netcdf4
-    - openmm ==7.0rc1 # temporarily test against stable version
+    - openmm ==7.0.1
     - mdtraj 1.5.1 # TODO remove me when segment_id bug is fixed (this is installed through openmoltools)
```
Comment (mpharrigan) [inline yank/meta.yaml]: likewise

File: yank/meta.yaml (hunk 1 above)
Comment (andrrizzi) [inline yank/meta.yaml]: Thanks for the info! I think it's too late for this release to test with the new version, but I will update the version in `dev`.

**Comment by andrrizzi** [comment]
> Congrats!

Thank you!

Looks like we are having the usual error for the missing c libraries on Linux: [`version 'GLIBC_2.7' not found (required by /anaconda/envs/_test/bin/antechamber_pvt)`](https://travis-ci.org/omnia-md/conda-recipes/jobs/145684009#L3597).

**Comment by andrrizzi** [comment]
Ok, everything seems to work now. Also, more comprehensive testing with `ambermini 15` on a Yank PR was successful. Merging.

**Pull request merged**